pkg/db/db_local/internal.go

```diff
@@ -14,24 +15,57 @@ import (
 	"github.com/turbot/steampipe/sperr"
 )
 
-/**
+// dropLegacyInternal looks for a schema named 'internal'
+// which has a function called 'glob' - and drops it
+func dropLegacyInternal(ctx context.Context, conn *pgx.Conn) error {
+	utils.LogTime("db_local.dropLegacyInternal start")
+	defer utils.LogTime("db_local.dropLegacyInternal end")
+
+	log.Println("[TRACE] counting legacy internal")
+
+	// we do a count here so that we don't have to deal with
+	// an antipattern of checking if the error is 'ErrNoRows'
+	// count will always yield a row - with a count of 0
+	legacySchemaCountQuery := `
```

Comment:
are we not going to check that glob is the only function and the only table is `connection_state` (or no table)?